### DISCUSSION TITLE
I have no reason to reject my application.

### DISCUSSION THREAD

**Discussion by Pixelek3** [community/community#113564] [GitHub Education]
### Hi everyone,

Hello, I'm trying to join the Github Eductaion, but my applications are rejected for no apparent reason: I've attached a screenshot below. I tried several solutions from the community but nothing works. At first, it rejected me because couldn't read the data (firstname, lastname and university name), but I corrected it and now I have something like this.

[image.png](https://github.zendesk.com/attachments/token/JIbvqaO030Ky3UGoLWQpljaIt/?name=image.png)

**Comment by samus-aran** [comment]
Hi @Pixelek3  👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by Pixelek3** [comment]
![Bez tytułu](https://github.com/community/community/assets/82956197/377ebeb8-ca03-4815-ba93-85d03353594c)
